isaric/parsers/ccp-drc.toml

```diff
@@ -964,7 +964,7 @@
   name = "oxygen_o2hb"
   phase = "study"
   date = { field = "hema_date" }
-  value = { field = "hemo", source_unit = { field = "hemo_type", values = { 1 = "g/L", 2 = "g/dL" } } }
+  value = { field = "hemo", unit = "g/dL", source_unit = { field = "hemo_type", values = { 1 = "g/L", 2 = "g/dL" } } }
```

Comment:
@pipliggins does this mean that the unit field response will still be used (most are 1 = g/L and the values appear to be within that range) or will the unit be applied as g/dL?

Reply:
@sadiekelly This means that in the parsed data file, all the hemoglobin values will be in units g/dL. Where in the raw data the units are marked as g/L, the value will be converted.